test/e2e/flowaggregator_test.go

```diff
@@ -162,11 +162,64 @@ type testFlow struct {
 	dstPodName string
 }
 
+func TestFlowAggregatorSecureConnection(t *testing.T) {
+	skipIfNotFlowVisibilityTest(t)
+	skipIfHasWindowsNodes(t)
+	options := []flowVisibilityTestOptions{
```

Comment:
I don't think you should conflate `flowVisibilityTestOptions` with your test case definition. Instead, maybe remove the `name` field from `flowVisibilityTestOptions`, and use a struct like this one for your test case definition:
```go
struct {
        flowVisibilityTestOptions
        name string
}
```

Reply:
Updated. Thanks.